frontend/packages/ceph-storage-plugin/src/components/create-storage-system/create-storage-system-steps/backing-storage-step/advanced-section.tsx

```diff
@@ -0,0 +1,80 @@
+import * as React from 'react';
+import { useTranslation } from 'react-i18next';
+import {
+  ExpandableSection,
+  ExpandableSectionProps,
+  FormGroup,
+  Select,
+  SelectOption,
+  SelectProps,
+  SelectVariant,
+} from '@patternfly/react-core';
+import { DeploymentType } from '../../../../constants/create-storage-system';
+import { WizardDispatch, WizardState } from '../../reducer';
+import './backing-storage.scss';
+
+export const AdvancedSection: React.FC<AdvancedSelectionProps> = ({
+  deployment,
+  isAdvancedOpen,
+  dispatch,
+}) => {
+  const { t } = useTranslation();
+  const [isOpen, setIsOpen] = React.useState(true);
+
+  const handleSelection: SelectProps['onSelect'] = (_event, selection: DeploymentType) => {
+    dispatch({ type: 'backingStorage/setDeployment', payload: selection });
+  };
+  const handleExpanadableToggling: ExpandableSectionProps['onToggle'] = (isExpanded: boolean) => {
+    dispatch({ type: 'backingStorage/setIsAdvancedOpen', payload: isExpanded });
+  };
+
+  const handleSelectToggling: SelectProps['onToggle'] = (isSelectExpanded: boolean) =>
+    setIsOpen(isSelectExpanded);
+
+  return (
+    <ExpandableSection
+      toggleText={t('ceph-storage-plugin~Advanced')}
+      onToggle={handleExpanadableToggling}
+      isExpanded={isAdvancedOpen}
+    >
+      <FormGroup label={t('ceph-storage-plugin~Deployment type')} fieldId="deployment-type">
+        <Select
+          id="deployment-type"
+          className="odf-backing-storage__selection--width odf-backing-storage__deployment-type"
+          placeholderText={t('ceph-storage-plugin~Select a deployment')}
+          aria-label={t('ceph-storage-plugin~Select deployment type')}
+          isCheckboxSelectionBadgeHidden
+          variant={SelectVariant.checkbox}
```

Comment:
```suggestion
          variant={SelectVariant.single}
```
Why do we need a checkbox variant here ? I assume only one item can be selected since it is select a deployment . 

All means block, file, object which includes MCG as well ..

IMO, This should rather be a small checkbox and completely remove the advance section as it does not add any value other than a small check unless you are adding more in further PRs.
`[x] MCG only `

if checked then the deployment will be with MCG only

Reply:
@yuvalgalanti ^^^

Reply:
Yes, it should be a radio button. I checked with Yuval.